tests/lib/rules/prop-types.js

```diff
@@ -1486,6 +1486,7 @@ ruleTester.run('prop-types', rule, {
         '  }',
         '}'
       ].join('\n'),
+      settings: {react: {flowVersion: '0.52.0'}},
```

Comment:
To be coherent with the React version we should default to the latest supported version.

But currently a breaking change (or deprecation) in React means a new major version bump for the plugin. Are breaking changes a common thing in Flow ? (I'm afraid this will force us to bump the major version to often).